hoomd/md/EvaluatorPairExpandedMie.h

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2009-2021 The Regents of the University of Michigan
+// This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+// Maintainer: joaander
+
+#ifndef __PAIR_EVALUATOR_ExpandedMie_H__
+#define __PAIR_EVALUATOR_ExpandedMie_H__
+
+#ifndef __HIPCC__
+#include <string>
+#endif
+
+#include "EvaluatorPairLJ.h"
+#include "hoomd/HOOMDMath.h"
+
+/*! \file EvaluatorPairExpandedMie.h
+    \brief Defines the pair evaluator class for Expanded Mie potentials
+*/
+
+// need to declare these class methods with __device__ qualifiers when building in nvcc
+// DEVICE is __host__ __device__ when included in nvcc and blank when included into the host
+// compiler
+#ifdef __HIPCC__
+#define DEVICE __device__
+#else
+#define DEVICE
+#endif
+
+//! Class for evaluating the Expanded Mie pair potential
+/*! <b>General Overview</b>
+
+    See EvaluatorPairSLJ and EvaluatorPairMie
+
+    <b>ExpandedMie specifics</b>
+
+    EvaluatorPairExpandedMie evaluates the function:
+    \f{eqnarray*}
+    V_{\mathrm{ExpMie}}(r)  = \left( \frac{n}{n-m} \right) {\left( \frac{n}{m}
+   \right)}^{\frac{m}{n-m}} \varepsilon \left[ \left( \frac{\sigma}{r-\Delta} \right)^{n} -
+                \left( \frac{\sigma}{r-\Delta} \right)^{m} \right] r \l (r_{\mathrm{cut}} +
+   \Delta)\\
+                = & 0 & r \ge (r_{\mathrm{cut}} + \Delta) \\
+    \f}
+    where  \f$ \Delta is specified by the user, conventionally calcualted as, \f$ \Delta = (d_i +
+   d_j)/2 - \sigma \f$ and \f$ d_i \f$ is the diameter of particle \f$ i \f$.
+
+    The ExpandedMie potential needs neither charge nor diameter. Two parameters are specified and
+   stored in a Scalar5. \a rep \a att \a npow \a mpow \a delta are stored in \a params.x \a params.y
+   \a params.z and \a params.w \a params.v respectively.
+
+
+    These are related to the standard lj parameters sigma and epsilon and the variable exponents n
+   and m by:
+    - \a rep = epsilon * pow(sigma,n) * (n/(n-m)) * power(n/m,m/(n-m))
+    - \a att = epsilon * pow(sigma,m) * (n/(n-m)) * power(n/m,m/(n-m))
+    - \a npow = n
+    - \a mpow = m
+    - \a delta = delta
```

Comment:
I would suggest using `repulsive`, `atractive`, `n_pow`, and  `m_pow` as it is more clear what each represents at a glance.